### ISSUE TITLE
Getting errors and vulnerabilities

### ISSUE THREAD

**Issue by ChristofJugel** [crsnbrt/keysim#53]
I tried to clone and run "npm install" However, there are multiple vulnerabilities. When I try to run, it says "./src/three/key/geometry.js
Attempted import error: 'Face3' is not exported from 'three' (imported as 'THREE')."

Does anyone have experience with that?

**Comment by lukedeboer** [comment]
Yeah I am getting the same thing. Looks there's a few tweaks that we need to make on our own to get it up and running.

**Comment by Alchard22** [comment]
> Yeah I am getting the same thing. Looks there's a few tweaks that we need to make on our own to get it up and running.

Hi Luke, I am trying to get the npm install to work also. Did you manage to find a work around?

Hope all is well :)
